tests/utils.go

```diff
@@ -2801,21 +2844,36 @@ func CheckForTextExpecter(vmi *v1.VirtualMachineInstance, expected []expect.Batc
 	}
 	defer expecter.Close()
 
-	resp, err := expecter.ExpectBatch(expected, time.Second*time.Duration(wait))
+	resp, err := ExpectBatchWithValidatedSend(expecter, expected, time.Second*time.Duration(wait))
 	if err != nil {
 		log.DefaultLogger().Object(vmi).Infof("%v", resp)
 	}
 	return err
 }
 
+func configureConsole(expecter expect.Expecter, prompt string) error {
+	batch := append([]expect.Batcher{
+		&expect.BSnd{S: "stty -inlcr\n"},
+		&expect.BExp{R: prompt},
+		&expect.BSnd{S: "sudo dmesg -n 1\n"},
+		&expect.BExp{R: prompt},
```

Comment:
wont you get a prompt even if the command failed ? 
so maybe worth to make sure all commands didnt fail ?